Automattic/wp-calypso
# Site Settings: Remove placeholder and related site existence checks

Some of our site settings still check for site existence. However, these checks are no longer necessary, as we won't be displaying the steps if the site isn't loaded yet. This also allows us to remove the placeholder that we displayed in that case.

This likely doesn't remove all the site existence checks, but hey - we have to start somewhere 😉 

#### Changes proposed in this Pull Request

* Remove the unnecessary site existence checks in site settings.
* Delete the placeholder component.
* Janitorial: Remove unnecessary prettier pragmas.
* Janitorial: Sort imports.
* Janitorial: Simplify functional components

#### Testing instructions

* Checkout this branch.
* Test all affected pages and make sure there are no regressions:
  * Guided transfer: `/settings/export/guided/:site` for Business .com sites.
  * Jetpack site disconnection flow.
  * General settings.
  * Export settings.
  * Import settings.
  * Discussion settings.
  * Security settings.
  * Writing settings.
* Verify the placeholder component is indeed no longer used.
